crates/lox-time/src/providers.rs

```diff
@@ -0,0 +1,278 @@
+#[macro_export]
```

Comment:
This macro does the heavy lifting for the `TryOffset` impls that cannot be blanket impls.